### PR TITLE
[Doc Update][DC and CEC]

### PR DESCRIPTION
**Pull request by v-ypalikila**

### CODE REVIEW

File: docs/custom-and-declarative-copilot.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## When to Choose Which?
+
+The decision to use a declarative or custom copilot hinges on several factors:
+
+|Declarative Copilots  |Custom Engine Copilots  |
+|---------|---------|
+|Orchestrator and foundation models managed by Microsoft     |   Bring your own Orchestrator and foundation models       |
+|For scenarios, which require a high level of focus or specialization.| For scenarios demanding extensive customization and control over the AI’s behavior|
+|Declarative copilots are suitable for developers with limited coding expertise or resources, as they offer a low-code development environment. | Custom copilots require pro-code skills and are ideal for teams with software development capabilities.|
```
Comment (aycabas) [inline docs/custom-and-declarative-copilot.md:39]: I think we shouldn't say "limited coding expertise" or low-code vs. pro-code. Why dont we rephrase this section as: "Developers who need to tailor Copilot for their scenarios can build Declarative copilots by simply declarating instructions in the app manifest. | Developers who need an advance customization for their Copilot experience can build Custom engine copilots by creating an AI chatbot."

File: docs/custom-and-declarative-copilot.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+|Create managed copilots with a guided low-code experience with  Copilot St.udio |   Create custom copilots with custom code and full control with pro-code using Azure AI studio or Visual Studio Code.  |
```
Comment (aycabas) [inline docs/custom-and-declarative-copilot.md:40]: No need pro-code, low-code comparison.

File: docs/custom-and-declarative-copilot.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+|You can start in low-code with a managed stack with Copilot Studio where you can:<br> * Quickly and securely build a solution based on your unique instructions, knowledge, and actions. <br><br> * Control the orchestration Of your large language model to fit your specific workflows. <br><br> * Deploy your copilots across websites, messaging channels, Microsoft Teams and as copilot extensions to serve employees, customers, and partners.    |You can enhance your own copilot with Azure AI Studio to: <br><br> * Use prebuilt models or train models using your data. <br><br> * Build and manage your AI applications. <br><br> * Use tools for creating, refining, evaluating, and maintaining the building blocks of your copilot.        |
```
Comment (aycabas) [inline docs/custom-and-declarative-copilot.md:41]: can extend skills and knowledge by utilizing Graph Connectors and plugins  | can extend skills and knowledge by utilizing Retrieval-Augmented Generation (RAG) and custom functions.

File: docs/custom-and-declarative-copilot.md (hunk 1 above)
Comment (v-ypalikila) [reply docs/custom-and-declarative-copilot.md:39]: updated

File: docs/custom-and-declarative-copilot.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+Custom Engine Copilots are another extension type that allows for the use of a custom Large Language Model (LLM) or Orchestrator, which is useful for scenarios requiring specific AI stacks or for publishing copilots to external users not on Microsoft. The Pro-Code Solution Custom Copilots cater to organizations with their own Copilot-like capabilities. Custom Engine Copilots are built using the Copilot SDK, which enables coding of instructions and action calls. Custom Copilots offer a higher degree of flexibility, allowing for the development of overrides to existing horizontal skills and the packaging of custom plugins, including message extensions and adaptive cards.
+
+## When to Choose Which?
+
+The decision to use a declarative or custom copilot hinges on several factors:
+
+|Declarative Copilots  |Custom Engine Copilots  |
+|---------|---------|
+|Orchestrator and foundation models managed by Microsoft     |   Bring your own Orchestrator and foundation models       |
+|For scenarios, which require a high level of focus or specialization.| For scenarios demanding extensive customization and control over the AI’s behavior|
```
[the remark this answers is not in the document]
Comment (v-ypalikila) [reply docs/custom-and-declarative-copilot.md:38]: updated

File: docs/custom-and-declarative-copilot.md (hunk 3 above)
Comment (v-ypalikila) [reply docs/custom-and-declarative-copilot.md:41]: Updated

File: docs/custom-and-declarative-copilot.md (hunk 2 above)
Comment (v-ypalikila) [reply docs/custom-and-declarative-copilot.md:40]: updated

**Comment by v-ypalikila** [comment]
The work in carried forward in PR https://github.com/MicrosoftDocs/m365copilot-docs-pr/pull/219.

**Pull request closed without merging**